dev-tools/validation/test_adr_review_status.py

```
@@ -0,0 +1,1970 @@
+        ):
+            with self.subTest(expected=expected), fixture_root() as fixture:
+                root = Path(fixture)
+                path = root / "design/architecture/decisions/README.md"
+                path.write_text(
+                    mutation(path.read_text(encoding="utf-8")),
+                    encoding="utf-8",
+                )
+                expect_failure(
+                    self,
+                    lambda: self.validator.supersession_index_rows(
+                        path, path.parent
```

Ruff (0.16.0)

[warning] Function definition does not bind loop variable `path`

(B023)